testdata/p4_16_errors/key-name.p4

```diff
@@ -34,4 +34,5 @@ control compute(inout hdr h) {
     }
 }
 
+/* P4TEST_IGNORE_STDERR */
```

Comment:
This is no longer needed if you pull my last outstanding PR; the include path should be ../p4_16_samples/arith-inline-skeleton.p4